src/main/webapp/appointments/create.jsp

```diff
@@ -8,104 +8,105 @@
-                                    <td><input type="email" name="email" value="${sessionScope.currentUser.email}" readonly></td>
-                                </tr>
-                            </table>
-                            <br>
-                            <br>
-                            <table border="1" cellpadding="5">
-                                <tr>
-                                    <th>Doctor</th>
-                                    <th>Slots</th>
-                                    <th>Selection</th>
-                                </tr>
-                                <c:forEach items="${requestScope.doctorSlots}" var="doctorToSlots">
-                                    <tr>
-                                        <td><c:out value="${doctorToSlots.x.firstName} ${doctorToSlots.x.surname}"/></td>
-                                        <td>
-                                            <select name="selectedSlot" required>
-                                                <c:forEach items="${doctorToSlots.y}" var="slot">
-                                                    <option value="${slot}/${doctorToSlots.x.ID}">${slot}</option>
-                                                </c:forEach>
-                                            </select>
-                                        </td>
-                                        <td>
-                                            <input required type="radio" name="staffID" value="${doctorToSlots.x.ID}">
-                                        </td>
-                                    </tr>
-                                </c:forEach>
-                            </table>
-                            <br>
-                            <br>
-                            <table border="1" cellpadding="5">
-                                <tr>
-                                    <th>Service Type</th>
-                                    <th>Date</th>
-                                </tr>
-                                <tr>
-                                    <td><select name="serviceType">
-                                        <option>public (NHS)</option>
-                                        <option>private</option>
+    <head>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=PT+Sans&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="${pageContext.request.contextPath}/styles/main.css"/>
+        <title>Patient Dashboard</title>
+    </head>
+    <body>
+        <div class="outer">
+            <div class="header">
+                <div class="innerDashboard">
+                    <img src="${pageContext.request.contextPath}/images/temp_logo.png" alt="logo" id="header-logo">
+                    <ul>
+                        <li><a>${requestScope.user}</a></li>
```

Comment:
Did we get this line working correctly now? I remember someone said it was dodgy.

Reply:
Yes it's working now, displays username